src/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,323 @@
+import React, {forwardRef} from 'react'
+import {
+  Text,
+  TextInput as Input,
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  TextProps,
+  ColorValue,
+  ReturnKeyTypeOptions,
+  ImageSourcePropType,
+  ImageResizeMode,
+  ImageStyle,
+  KeyboardTypeOptions,
+} from 'react-native'
+import {metrics} from '../helpers/metrics'
+import {Icon} from '../Icon/Icon'
+import styled from 'styled-components/native'
+import {colors} from '../helpers/colors'
+
+export interface TextInputProps {
```

Comment:
can you extend TextInputProps from react-native?

import {TextInputProps} from 'react-native';

Reply:
done